apps/notifications/signals.py

```diff
@@ -12,8 +12,10 @@
 @receiver(post_save, sender=Action)
 def send_notification(sender, instance, created, **kwargs):
     action = instance
-    if action.verb == Verbs.ADD.value and (
-            action.obj_content_type.model_class() is IdeaSketch
-            or action.obj_content_type.model_class() is Proposal
-    ):
-        emails.SubmitNotification.send(action.obj)
+    if action.verb == Verbs.ADD.value:
+
+        if (action.obj_content_type.model_class() is IdeaSketch
+                or action.obj_content_type.model_class() is Proposal):
+            emails.SubmitNotification.send(action.obj)
+        else:
```

Comment:
Should we add a more specific check here? This will now also enable CreatorNotifications if a Collaborator creates a JourneyEntry. Maybe that is what we want maybe not. And there is always the danger of someone adding another actionable that will result in even more notifications.